packages/financial-templates-lib/src/helpers/acrossFeesCalculator.ts

```diff
@@ -0,0 +1,68 @@
+// This Util calculates the across realized LP fees. TODO: add a link to chase's notebooks once they are production ready
```

Comment:
Yeah would be useful to have plain text math formulas to reference when checking your implementation

Reply:
I created a gist with this file for now until we have the official version from chase